requirements/fabric/docs.txt

```diff
@@ -0,0 +1,4 @@
+-r ../docs.txt
+
+lai-sphinx-theme
+tensorboardX>=2.2, <=2.5.1  # min version is set by torch.onnx missing attribute
```

Comment:
do we need tensorboard for docs?

Reply:
just refering what is there, but I am fine to switch it :)

Reply:
```suggestion
tensorboard
```